.pre-commit-config.yaml

```
@@ -47,6 +47,7 @@ repos:
     - --django-settings-module=mysite.settings
     additional_dependencies:
     - pylint_django[with_django] >= 2.5.3
+    - coverage
```

It's usually nicer to have these entries sorted:
```suggestion
    - coverage
    - pylint_django[with_django] >= 2.5.3
```